src/Sandbox.php

```diff
@@ -740,4 +740,18 @@ public function usesProxy(): bool
     {
         return !empty($this->getUrl()) && preg_match('/^https:\/\/[^.]+\.hub\./ui', $this->getUrl()) === 1;
     }
+
+    public function getJupyterApiUrl(): ?string
+    {
+        if (!in_array($this->getType(), self::JUPYTER_TYPES)) {
+            return null;
+        }
+
+        // Remove '/lab` suffix from sandbox url
+        $url = preg_replace('/\/lab$/', '', (string) $this->getUrl());
+        if (empty($url) or empty($this->getPassword())) {
```

Comment:
LOL to `or` je hodne velkej relic :smile:

Reply:
https://github.com/keboola/sandboxes-api-php-client/pull/83/commits/a2c897d07aca39a1fb393da98a872106f260f14c